main_2024.py

```diff
@@ -8,6 +8,7 @@
 import multiprocessing as mp
 import pathlib
 import queue
+from enum import Enum
```

Comment:
take a look at https://uwarg-docs.atlassian.net/wiki/spaces/CV/pages/2253226033/Python+Style+Convention#Imports for import style convention (make sure imports are in alphabetical order within their section)